crest/Assets/Crest/Crest/Shaders/SdfShared.hlsl

```diff
@@ -0,0 +1,20 @@
+// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// Setting this to 0.0 means that gometry at exactly the origin won't be handled
```

Comment:
just to understand, what will the artifact be? is this only if the texel center is exactly at the origin?

it seems like a moderately possible scenario, but if the impact is low then its fine. i was trying to think of a better magic number, but didnt come up with anything (safe) so far!